tests/robustness/failpoint/failpoint.go

```diff
@@ -570,3 +574,133 @@ func (f killAndGofailSleep) Available(config e2e.EtcdProcessClusterConfig, membe
 	}
 	return memberFailpoints.Available(f.failpoint)
 }
+
+type memberReplace struct{}
+
+func (f memberReplace) Inject(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster) error {
+	memberId := rand.Int() % len(clus.Procs)
+	member := clus.Procs[memberId]
+	var endpoints []string
+	for i := 1; i < len(clus.Procs); i++ {
+		endpoints = append(endpoints, clus.Procs[(memberId+i)%len(clus.Procs)].EndpointsGRPC()...)
+	}
+	cc, err := clientv3.New(clientv3.Config{
+		Endpoints:            endpoints,
+		Logger:               zap.NewNop(),
+		DialKeepAliveTime:    50 * time.Second,
+		DialKeepAliveTimeout: 100 * time.Millisecond,
+	})
+	if err != nil {
+		return err
+	}
+	defer cc.Close()
+	memberID, found, err := getID(ctx, cc, member.Config().Name)
+	if err != nil {
+		return err
+	}
+	if !found {
+		t.Fatal("Member not found")
+	}
+	// Need to wait health interval for cluster to accept member changes
+	time.Sleep(etcdserver.HealthInterval)
```

Comment:
Do we need to have some further verification here that the member is actually operating successfully?

Reply:
No, health verification is done before failpoint injection. Member is already healthy and serving requests. Membership changes just require this additional wait.